OrbitCore/StringManager.h

```diff
@@ -16,8 +16,9 @@ class StringManager {
   StringManager() = default;
 
   bool AddIfNotPresent(uint64_t key, std::string_view str);
-  std::optional<std::string> Get(uint64_t key);
-  bool Contains(uint64_t key);
+  bool AddOrReplace(uint64_t key, std::string_view str);
+  std::optional<std::string> Get(uint64_t key) const;
+  bool Contains(uint64_t key) const;
```

Comment:
[[nodiscard]] for last 2 functions

Reply:
Let's discuss the use of [[nodiscard]] with the team.

Reply:
Done.